src/styles/calendar.scss

```diff
@@ -0,0 +1,49 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  counter-reset: day-number 0;
+  display: flex;
+  flex-wrap: wrap;
+  width: calc((#{$day-size} * 7) + (#{$gap} * 6) + #{$padding} * 2);
+  padding: $padding;
+  gap: $gap;
+}
+
+h1 {
+  display: none;
+}
+
+.calendar__day {
+  width: $day-size;
+  height: $day-size;
+  background-color: $bg-color-default;
+  border: $border-width solid $border-color;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+
+  // Псевдоелемент для чисел днів
+  &::before {
+    content: counter(day-number);
+    counter-increment: day-number;
+    font-size: 30px;
+    font-family: Arial, sans-serif;
+    color: $text-color;
+    position: absolute;
+  }
+
+  &:hover {
+    background-color: $bg-color-hover;
+    cursor: pointer;
+    transform: translateY(-20px);
+    transition:
+      transform $animation-duration,
+      background-color $animation-duration;
+  }
```

Comment:
The transition property should be set on the element itself, not on the hover state. This way, the transition will be smooth both when the element is hovered over and when the mouse leaves the element. Currently, the transition only works when the element is hovered over, but it instantly jumps back to its original state when the mouse leaves.